src/Chunk.ts

```diff
@@ -860,11 +860,11 @@ export default class Chunk {
 		options: OutputOptions,
 		existingNames: Record<string, any>
 	): string {
-		const hash = sha256();
+		const hash = createHash('sha256');
 		hash.update(
 			[addons.intro, addons.outro, addons.banner, addons.footer].map(addon => addon || '').join(':')
 		);
-		hash.update(options.format);
+		hash.update(options.format as string);
```

Comment:
Unfortunately, this typecast is needed because we reuse the outward facing option type internally. I hope at some point the logic can be rewritten so that internally, we use a normalized type where format always exists, but this will take time.